docs/FAQ.md

```diff
@@ -223,6 +223,114 @@ Questions about VictoriaMetrics can be asked via the following channels:
 File bugs and feature requests [here](https://github.com/VictoriaMetrics/VictoriaMetrics/issues).
 
 
+### Where I can find information about Multi-Tenancy?
+
+You can read about Multitenancy [here](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#multitenancy). Note that only a [cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html) version supports Multi-Tenancy.
+
+
+
+### Will having multiple tenants affect my performance? Is this data written on disk separately or in one place?
+
+If you take the most challenging case  (multi-tenancy for the same set of metrics) it will require more disk space, more RAM for caches and more CPU for data processing. There should be linear growth for each additional tenant. But, in general, the overhead from additional tenants should be negligible because the metrics should be differentiated by labels. When the  tenant-id is added to the metric ID, and everything else - indexes, disk placement, remains the same.
+
+
+### Is it possible to delete a tenant by deleting its metrics?  
+
+Yes, it’s possible. Also we have this [issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1111).
+
+
+### How to set a memory limit for vmstorage ? Cluster/SingleNode version
+
+VictoriaMetrics provides flags to control the size of internal buffers and caches. But these are soft limits and the total process memory usage may exceed it. Hard limits may be enforced only by OS ([cgroups](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/sec-relationships_between_subsystems_hierarchies_control_groups_and_tasks)) or virtualization tools ([docker](https://docs.docker.com/config/containers/resource_constraints), [k8s](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers)).
+
+
+### Can I use VictoriaMetrics instead of an all Prometheus Stack (Prometheus, Alertmanager, PushGateway) or are there some things that can't be replaced?
+
+Yes, you can replace your entire Prometheus stack with VictoriaMetrics
+
+
+### I have two Prometheus replicas in my k8s cluster that write data into VictoriaMetrics and therefore see dual data from node_exporter’s in my Grafana dashboard. I have tried to use  `-dedup.minScrapeInterval: 10s`  parameter for vmagent https://docs.victoriametrics.com/#deduplication but my Pod was disconnected because it didn’t know this flag.
```

Comment:
"two Prometheus" => "two Prometheis"

The question is too specific. I don't know if we need it at all. We should either make it more general (how to deduplicate) or improve deduplication docs.